### PR TITLE
feat: Add task for running peribolos on push event

### PR DESCRIPTION
**Pull request by mimotej**
Part of: #10 
This is task for running peribolos. Task gets token from a secret and uses it to run peribolos.

### CODE REVIEW

File: pipelines/base/peribolos-run.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: peribolos-run
+spec:
+  params:
+  - name: REPO_NAME
+    type: string
+    default: '.github'
+  - name: CLIENT_ID
```
Comment (tumido) [inline pipelines/base/peribolos-run.yaml:10]: This should be **Installation ID**, client ID is a different thing - unique per application, shared for all installations.

image

**Review by tumido — CHANGES_REQUESTED** [review]
Nit, naming of a param is off, otherwise LGTM

File: pipelines/base/peribolos-run.yaml (hunk 1 above)
Comment (mimotej) [reply pipelines/base/peribolos-run.yaml]: Fixed, forgot there is a difference :smile:.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**